cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -74,7 +108,7 @@ spec:
         imagePullPolicy: IfNotPresent
         command:
         - /usr/local/bin/kube-proxy
-        - --kubeconfig=/var/lib/kube-proxy/kubeconfig.conf
+        - --config=/var/lib/kube-proxy/config.conf
```

Comment:
You should not change the v1.8 version of the manifest, that one should stay the same

Reply:
done